.github/workflows/azure_ci_check.yml

```diff
@@ -0,0 +1,72 @@
+name: Azure CI
+
+on:
+  pull_request:
+    types: [ opened, reopened, synchronize, edited ]
+  issue_comment:
+    types: [ created, edited, deleted ]
+
+jobs:
+  check-azure-ci-report:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Get last commit hash
+        id: last_commit
+        uses: actions/github-script@v5
+        with:
+          script: |
+            const pr = context.payload.pull_request;
+            const lastCommitHash = pr.head.sha;
+            console.log(`Last commit hash: ${lastCommitHash}`);
+            // Set the output variable to be used in subsequent step
+            core.setOutput("last_commit_hash", lastCommitHash);
+
+      - name: Check Azure CI report in PR comment
+        uses: actions/github-script@v5
+        with:
+          script: |
+            const lastCommitHash = '${{ steps.last_commit.outputs.last_commit_hash }}'
+            const botUsername = 'hudi-bot';
+            const contentToCheck = 'specific content';
```

Comment:
is this used?

Reply:
Good catch.  Going to remove `contentToCheck`